coverage/clover.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What does this file do?

Reply:
yea how not to push these coverage files? shall i git checkout coverage every time before git commit?

Reply:
`git reset <filename>` can do. In this case, do `git reset coverage` since we want to unstage all the files in `coverage` folder

Reply:
i think to `git rm coverage -r -f` to hard remove the coverage files worked well